webapi/appsettings.json

```
@@ -61,15 +61,11 @@
     // Set RelevancyThreshold to a value between 0 and 1 if using the SequentialPlanner or Stepwise planner with gpt-3.5-turbo.
     // Ignored when Planner:Type is "Action"
     "RelevancyThreshold": "0.25",
-    // Whether to allow missing functions in the plan on creation then sanitize output. Functions are considered missing if they're not available in the planner's kernel's context.
-    // If set to true, the plan will be created with missing functions as no-op steps that are filtered from the final proposed plan.
-    // If this is set to false, the plan creation will fail if any functions are missing.
-    "MissingFunctionsError": {
-      "AllowSkips": "true",
-      "MaxRetriesAllowed": "3" // Max retries allowed on MissingFunctionsError. If set to 0, no retries will be attempted.
+    "PlannerError": {
+      "AllowRetries": "true",
+      "AllowMissingFunctions": "true",
```

Again, I believe we should keep a comment to explain what AllowMissingFunctions is

---

+1 on adding the comment. Would recommend renaming this to 
```suggestion
      "SkipMissingFunctionsError": "true",
```

---

`AllowMissingFunctions` is defined in the `SequentialPlannerConfig`. I think it's better to align the names. I will add a comment.

---

It's not the same behavior. In the `SequentialPlannerConfig`, AllowMissingFunctions allows the missing functions in the final plan it returns, whereas enabling this flag here means allowing the missing functions and then removing them from the final plan, so I think `Skip` is more indicative of what the user is setting here.